dovetail number 006 (#1459)
carpentries/carpentries.org: _posts/2022/08/2022-08-24-dovetail-006.md

our new Director of Technology. During this time, I was able to tackle some
superficial aspects of the workbench itself. 

In this issue of The Dovetail, I wanted to reiterate the purpose of the Beta
Phase of The Workbench (which is still on hold) because it bears repeating. We
are going to be transitioning all of our lessons to use The Carpentries
Workbench starting in 2023, and the Beta Phase is here to prepare the community
for this day by doing the following:

1. Creating awareness about The Workbench and [how workbench lessons differ from styles lessons](https://carpentries.github.io/workbench/transition-guide.html).
2. Providing a risk-free sandbox for maintainers to and instructors work with
   The Workbench version of lessons before full transition.

If we have the above two points, we know that we can proceed with the transition
of our lessons in 2023. However, The Workbench Beta Phase is designed to do
more than simply making the community aware of the changes; we want to make 
sure The Workbench is _thriving_ in the hands of the community. That is, we want
to: 

3. Make sure that the tool actually meets the needs and wants of the community. 

> 4. Create champions future contributors to The Workbench.

Author: This is a good point. Its less of a goal and more of a side-effect from transparently fixing issues and crediting community members in places where they were having trouble.  I've updated the wording to reflect this aspiration.

```suggestion
4. Create champions of, and inspire future contributors to The Workbench.
```

When I piloted the [Alpha Test of The Workbench]({{ site.url

> }}/blog/2021/07/infrastructure-testing), my goal was to simply make sure that

Reviewer: ```suggestion
}}/blog/2021/07/infrastructure-testing), my goal was to make sure that
```

Don't undersell yourself - it didn't look simple to me!

it could be installed and work as expected for members of our community. The
move into Beta testing means that I am confident that The Workbench is indeed
stable enough to host our lessons (i.e. [the official documentation for The
Workbench](https://carpentries.github.io/sandpaper-docs/) has been continuously
hosted for the last year and a half, since 2021-02-09), and I am ready for
early adopters to try it out and really tell me how they _feel_ about The
Workbench.

This can only be really acheived with directed and specific feedback, which is
why I've given the [Beta Phase a fairly rigid structure in three stages]({{ site.url }}/blog/2022/05/workbench-beta/#beta-logistics).

> During each stage, I will be asking Maintainers and Instructors to choose from
> and try out different, specific, tasks and give feedback in GitHub Issue on The
> Workbench repository. For an extra level of detail, I am hoping that I can find

Reviewer: ```suggestion
During each stage, I will be asking Maintainers and Instructors to
try out different, specific, tasks and give feedback in GitHub Issues on 
[The Workbench repository](https://github.com/carpentries/workbench).
For an extra level of detail, I am hoping that I can find
```

> some maintainers or instructors to provide [Friction Logs](https://github.com/carpentries/workbench/discussions/2),

Reviewer: ```suggestion
some Maintainers or Instructors to provide [Friction Logs](https://github.com/carpentries/workbench/discussions/2),
```

which will give me a step-by-step view into a specific process so that the

> exact cause of the friction can be identified and smoothed out.

Reviewer: ```suggestion
exact causes of friction can be identified and smoothed out.
```

---
layout: page
authors: ["Zhian N. Kamvar"]
teaser: "Hello Again"
title: "The Dovetail #6: Updates from The Carpentries Workbench"
date: 2022-08-24
time: "09:00:00"
tags: ["Lesson Infrastructure", "Community", "Carpentries Workbench", "Beta", "Dovetail"]
---

> This is the sixth post in a series that we are calling "The Dovetail."
> In this series, we aim to keep members of The Carpentries community abreast of
> the current news for [The Carpentries Workbench](https://carpentries.github.io/workbench). 

Reviewer: ```suggestion
This is the sixth post in [a series that we are calling "The Dovetail",
about the transition to The Carpentries Workbench](https://carpentries.org/posts-by-tags/#blog-tag-dovetail).
In this series, we aim to keep members of The Carpentries community abreast of
the current news about [the Workbench](https://carpentries.github.io/workbench). 
```

> 

Reviewer 2: We should add a sentence and link around this area to the rest of the posts if people are interested

Author: I agree. Do you want to add this before you merge?

Author: It looks like toby made the changes. Feel free to merge

If you are interested in participating in discussions around The Carpentries
Workbench, head over to our GitHub Discussions forum: <https://github.com/carpentries/workbench/discussions>

---

A lot has happened since the last time I provided an update (2022-07-20), just

> not that much on The Workbench side of things. I atteneded and [presented at RStudio conference 2022](https://zkamvar.netlify.app/talk/carpentries-rstudio-2022/),

Reviewer 2: ```suggestion
not that much on The Workbench side of things. I attended and [presented at the RStudio conference 2022](https://zkamvar.netlify.app/talk/carpentries-rstudio-2022/),
```

> took a short staycation to decompress (with a broken aircon, no less!),

Reviewer 2: ```suggestion
took a short "staycation" to decompress (with a broken aircon, no less!),
```
